WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -99,6 +107,15 @@ class WPMainActivityViewModel @Inject constructor(
             .distinctUntilChanged()
             .map { Event(it) } as LiveData<Event<List<FocusPointInfo>>>
 
+    val isMultipleSiteAvailable: Boolean
+        get() = mSiteStore.sitesCount > ONE_SITE
+
+    val firstSite: SiteModel?
```

Comment:
As the site returned by `firstSite` variable is the one selected by default in the site picker screen, can we rename this to `selectedSite`?. WDYT? 🤔

Reply:
At this point of time, there are actually no selected sites. So naming the property as first site makes it clear that we are sending first site, which will be considered as selected site in site picker activity.
```kotlin
ActivityLauncher.showSitePickerForResult(this, mViewModel.getFirstSite()); // It seems clear that we are sending first site as selected site
ActivityLauncher.showSitePickerForResult(this, mViewModel.getSelectedSite());
```
If you still feel, it makes more sense when it is changed as selected site, I can do it. WDYT?

Reply:
>At this point of time, there are actually no selected sites. So naming the property as first site makes it clear that we are sending first site, which will be considered as selected site in site picker activity.

@SowmiaS 👋 I think you are right. Let's keep the naming as `firstSite` itself. 🙌